TestVectorsAwsCryptographicMaterialProviders/runtimes/python/fuzz_generator.py

```diff
@@ -194,7 +227,7 @@ def create_kms_based_key_description(draw, keyring_type: str, kms_key: str, requ
     # Map keyring types to their underlying types and keys
     keyring_config = {
         "kms": {"type": "aws-kms", "key": kms_key},
-        "aws-kms-mrk-aware": {"type": "aws-kms-mrk-aware", "key": draw(st.sampled_from(MRK_KEYS))},
+        #"aws-kms-mrk-aware": {"type": "aws-kms-mrk-aware", "key": draw(st.sampled_from(MRK_KEYS))},
```

Comment:
Comments like this just creates a lot of noise in the code. You can add a TODO if its not there yet. 

```suggestion
```